docs/BesuProxyModel.md

```diff
@@ -0,0 +1,43 @@
+
+**Testing Avalon Proxy model with Hyperledger Besu**
+1.  Install ``truffle`` using the command
+    ```
+    npm install -g truffle
+    ```
+    This requires Requires ``NodeJS v8.9.4 or later`` preinstalled.
+
+2.  Set the environment variable ``TCF_HOME`` to the Avalon root directory
+
+3.  Start the Ethereum network with 2 nodes and 2 eth-signers. To bring up the network, do the following
+    ```
+    cd $TCF_HOME/docs/dev-environments/besu
+    docker-compose up -d
+    ```
+       This will start the Ethereum network locally.
+
+4.  Now you need to create a truffle project and deploy contracts onto the Ethereum network
+       ```
+        mkdir ./my_project
+        cd my_project
+        cp $TCF_HOME/sdk/avalon_sdk/ethereum/contracts/proxy-model/*.sol ./contracts/
+        cp $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts/truffle-config.js ./
+        cp $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts/2_deploy_contracts.js ./migrations/
+        truffle migrate --network avalon
```

Comment:
Running this results in this error:
```
avalon@avalonhost:~/my_project$ truffle migrate --network avalon
You can improve web3's peformance when running Node.js versions older than 10.5.0 by installing the (deprecated) scrypt package in your project

Compiling your contracts...
===========================
> Compiling ./contracts/WorkOrderRegistry.sol
> Compiling ./contracts/WorkerRegistry.sol
> Artifacts written to /home/avalon/my_project/build/contracts
> Compiled successfully using:
   - solc: 0.5.15+commit.6a57276f.Emscripten.clang

Error: Could not find built Migrations contract: Could not find artifacts for Migrations from any sources
    at Object.lastCompletedMigration (/usr/local/lib/node_modules/truffle/build/webpack:/packages/migrate/index.js:184:1)
    at Promise (/usr/local/lib/node_modules/truffle/build/webpack:/packages/migrate/index.js:220:1)
    at new Promise (<anonymous>)
    at Object.needsMigrating (/usr/local/lib/node_modules/truffle/build/webpack:/packages/migrate/index.js:217:1)
    at runMigrations (/usr/local/lib/node_modules/truffle/build/webpack:/packages/core/lib/commands/migrate.js:250:1)
    at Contracts.compile.then (/usr/local/lib/node_modules/truffle/build/webpack:/packages/core/lib/commands/migrate.js:218:1)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Truffle v5.1.14 (core: 5.1.14)
Node v8.10.0
avalon@avalonhost:~/my_project$ ls
build  contracts  migrations  truffle-config.js
avalon@avalonhost:~/my_project$ ls *
truffle-config.js

build:
contracts

contracts:
WorkerRegistry.sol  WorkOrderRegistry.sol

migrations:
2_deploy_contracts.js
avalon@avalonhost:~/my_project$ 
```

I am going to try the old version of the command (that does not copy these files).  That is:
```
cd $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_project
truffle migrate --network avalon
```